docs/tutorials/basics.ipynb

```diff
@@ -6,7 +6,7 @@
     "id": "7FV_m2u2Hny8"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    circuit.append(cirq.H(q) for q in cirq.LineQubit.range(3))
</pre><p>Is it confusing to start with a case where you are passing a generator?  Maybe in this case an explicit list [H(q0), ...] is better?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5406/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Great point. Added some additional options/context to this.